CHANGELOG.md

```diff
@@ -25,6 +25,7 @@ Copy and pasting the git commit messages is __NOT__ enough.
 ## [1.0.7] # 2023-03-04
 ### Added
 - Added objective, scenario description, and trained agent performance, to the Driving Smarts 2022 benchmark documentation.
+- Added support for the [Argoverse 2 Motion Forecasting Dataset](https://www.argoverse.org/av2.html#forecasting-link) (see `scenarios/argoverse`).
```

Comment:
Should this changelog entry be under unreleased?

Reply:
I would suggest rebasing the branch because there is now a "1.0.8" section.

Reply:
It got moved after merging master in. I've moved it back to the unreleased section.